index.html

```diff
@@ -135,6 +143,86 @@ <h2>Project name goes here</h2>
       </div>
     </section>
 
+    <section class="desktop-work">
+      <h2 class="desktop-projects">Projects</h2>
+      <div class="my-first-project">
+        <div class="project-overlay"></div>
+        <h2>Project name goes here</h2>
+        <ul>
+          <li>HTML/CSS</li>
+          <li>Ruby on Rails</li>
+          <li>JavaScript</li>
+        </ul>
+        <button type="button">
+          See this project
+          <img src="assets/desktop-version/Union.png" alt="right arrow icon" />
+        </button>
+      </div>
+      <div class="my-second-project">
+        <h2>Project name goes...</h2>
+        <ul>
+          <li>HTML/CSS</li>
+          <li>Ruby on Rails</li>
+          <li>+1</li>
+        </ul>
+        <button type="button">
+          See this project
+          <img src="assets/desktop-version/Union.png" alt="right arrow icon" />
+        </button>
+        <div class="overlay-2"></div>
+      </div>
+    </section>
+
+    <section class="desktop-about">
+      <div class="technologies-languages">
+        <h2>Languages</h2>
+        <ul>
+          <li>JavaScript (ES6+)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+        <div class="hashtag-desktop"></div>
+      </div>
+      <div class="technologies-framework">
+        <h2>Frameworks</h2>
+        <ul>
+          <li>React.js</li>
+          <li>Ruby on Rails</li>
+          <li>RSpec</li>
+          <li>Capibara</li>
+        </ul>
+        <div class="framework-desktop"></div>
+      </div>
+      <div class="technologies-skills">
+        <h2>Skills</h2>
+        <ul>
+          <li>Database Management</li>
+          <li>CLI</li>
+          <li>API Design</li>
+          <li>Version Control</li>
+          <li>Web Development</li>
+        </ul>
+        <div class="skills-desktop"></div>
+      </div>
+      <div class="grid">
+        <div class="images"></div>
+      </div>
+    </section>
+
+    <aside class="about-me-desktop">
+      <h2>About</h2>
+      <p>
+        I can help you build a product , feature or website Look through <br />
+        some of my work and experience! If you like what you see and <br />
```

Comment:
- [OPTIONAL] You will notice that the about section does not have a background image, I would implore you to separate the aspect of the image and the about section in your project also.

### Figma
![image](https://user-images.githubusercontent.com/86200356/236645692-bd149424-c2e1-468d-8585-fd1df01713ed.png)
 
### Your design
![image](https://user-images.githubusercontent.com/86200356/236645707-6eef99c6-ba7a-4507-8d84-0f164409ab7f.png)